package.json

```diff
@@ -26,47 +24,59 @@
     "network",
     "browser"
   ],
+  "main": "dist/umd.js",
+  "module": "dist/esm.js",
+  "types": "dist/types/index.d.ts",
+  "files": [
```

Comment:
What is this "files" section used for?

Reply:
It's like npmignore or gitignore, just the other way around, it says what you want in the package. So that we don't include source files, docs and other bloat in other peoples node_module folders.

Reply:
Thanks! I see: https://docs.npmjs.com/files/package.json#files
Wouldn't it be usefull to also include the `src` folder so that other modules (like vis-timline and vis-network) dirctly could require classes. I guess this also would be neccessary to use tree-shaking from rullup!?

Reply:
Nope. Rollup will use the module field, which is ES2015 module where tree shaking works just fine.